README.md

```diff
@@ -66,6 +66,10 @@ $ ./faros_event.sh CD -k "<faros_api_key>" \
     --deploy_end_time "1626804357000"
 ```
 
+Every time a artifact is deployed to an environment, you can send an event.
+
+![When to send an event](resources/events.png)
```

Comment:
we use `deploy_env` as the place holder name for the environment. Does it make sense to use that in the picture instead of `Environment`?